src/components/SurveyDetails/index.js

```diff
@@ -0,0 +1,133 @@
+import React, {Component, Fragment} from 'react';
+import PropTypes from 'prop-types';
+import {Text} from 'react-native';
+import {includes, some} from 'lodash';
+import {connect} from 'react-redux';
+import {
+    Button,
+    LoadingIndicator,
+    Title,
+    Row,
+    Col,
+    TextStrong
+} from '@indec/react-native-commons';
+import {Alert} from '@indec/react-native-commons/util';
+
+import {requestSurvey, requestCloseSurvey} from '../../actions/survey';
+import matchParamsIdPropTypes from '../../util/matchParamsIdPropTypes';
+
+const responseAnswers = ({
+    YES: 1,
+    NO: 2
+});
+
+const showForceCloseButton = survey => {
+    if (!survey.dwellings) {
+        return false;
+    }
+    if (!survey.dwellings[0].response || survey.dwellings[0].response === responseAnswers.NO) {
+        return false;
+    }
+    if (includes([2, 3], survey.dwellings[0].responseCause)) {
+        return true;
+    }
+
+    const hasHouseholdWithNoAnswer = some(survey.dwellings, dwelling =>
+        some(dwelling.households, household =>
+            household.response === responseAnswers.NO
+        )
+    );
+
+    if (hasHouseholdWithNoAnswer) {
+        return true;
+    }
+
+    const hasMemberdWithNoAnswer = some(survey.dwellings, dwelling =>
+        some(dwelling.households, household =>
+            some(household.members, member =>
+                member.response === responseAnswers.NO
+            )
+        )
+    );
+    return !!hasMemberdWithNoAnswer;
+};
+
+class SurveyDetails extends Component {
+    static propTypes = {
+        requestSurvey: PropTypes.func.isRequired,
+        survey: PropTypes.shape({}),
+        forceCloseSurvey: PropTypes.func.isRequired,
+        match: matchParamsIdPropTypes.isRequired
+    };
+
+    static defaultProps = {
+        survey: null
+    };
+
+    componentDidMount() {
+        this.props.requestSurvey(this.props.match.params.id);
+    }
+
+
+    forceClosePressed(surveyId) {
```

Comment:
sacar este param y usar el this.props.match.params.id

Reply:
Done x 2